hyrax.gemspec

```diff
@@ -81,7 +81,7 @@ SUMMARY
   spec.add_dependency 'samvera-nesting_indexer', '~> 2.0'
   spec.add_dependency 'select2-rails', '~> 3.5'
   spec.add_dependency 'signet'
-  spec.add_dependency 'tinymce-rails', '~> 4.1'
```

Comment:
Do we want to pin this to a minor version?  I don't consider that a blocker.  I see that we have lots of depedency's pinned to minor version.

Reply:
note that `~> 4.1` in ruby gem spec means "any 4.x at least 4.1", it's not necessarily what you'd call "pinning to a minor version". Was just passing by and wanted to point this out becuase it seems to be frequently confused and is kind of important in your gemspec.  If you actually wanted to pin to "4.1.x", you can say `"~> 4.1.0"`, or `">= 4.1.0", "< 4.2"`, etc.